mvnw.cmd

```diff
@@ -1,145 +1,145 @@
-@REM ----------------------------------------------------------------------------

```

Comment:
Yes, that commit is the side-effect of adding `.gitattributes`.  It looked like `mvnw.cmd` should have DOS line endings, so I marked it that way in `.gitattributes` and git converted it.  I could revert this change if that wasn't the right decision.